R/log_lik.R

```diff
@@ -320,8 +320,7 @@ ll_args.stanreg <- function(object, newdata = NULL, offset = NULL, m = NULL,
     }
     data <- data.frame(y, x)
     draws$beta <- stanmat[, colnames(x), drop = FALSE]
-    patt <- if (length(unique(y)) == 2L) "(Intercept)" else "|"
-    zetas <- grep(patt, colnames(stanmat), fixed = TRUE, value = TRUE)
+    zetas <- grep("|", colnames(stanmat), fixed = TRUE, value = TRUE)
```

Comment:
I think we can get rid of `patt` here because length(unique(y)) can never legitimately be 2 if `is_polr(object)` is TRUE. When you fit a binary model with `stan_polr()`, `is_polr(object)` is FALSE, so this part of the `log_lik()` code isn't used.

Reply:
OK